Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/dccwalletinfo/calculation/CclDefaultInputParameters.kt

```diff
@@ -8,20 +8,20 @@ import org.joda.time.format.ISODateTimeFormat
 import timber.log.Timber
 import java.util.Locale
 
-internal fun getDefaultInputParameters(
+fun getDefaultInputParameters(
```

Comment:
What about moving this file from the `dccwalletinfo` package into the `ccl` root package, since its now not only used by `dccwalletinfo` anymore?